packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -93,7 +94,8 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
         @inject(OpenerService) protected readonly openerService: OpenerService,
         @inject(FileNavigatorFilter) protected readonly fileNavigatorFilter: FileNavigatorFilter,
         @inject(WorkspaceService) protected readonly workspaceService: WorkspaceService,
-        @inject(WorkspacePreferences) protected readonly workspacePreferences: WorkspacePreferences
+        @inject(WorkspacePreferences) protected readonly workspacePreferences: WorkspacePreferences,
```

Comment:
Please use `property` injections over `constructor` injections.
The addition of new constructor injections are a [breaking change](https://github.com/theia-ide/theia/wiki/Coding-Guidelines#property-injection).

Reply:
Should I also tidy the others up while I am there?

Reply:
> Should I also tidy the others up while I am there?

No, only the new ones :) Adding or removing the constructor injections would be breaking changes.